src/wagi_config.rs

```diff
@@ -12,11 +16,11 @@ pub struct WagiConfiguration {
     pub log_dir: PathBuf,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub enum HandlerConfigurationSource {
     ModuleConfigFile(PathBuf),
     StandaloneBindle(PathBuf, bindle::Id),
-    RemoteBindle(url::Url, bindle::Id),
+    RemoteBindle(bindle::Id, BindleConnectionInfo),
```

Comment:
Reverse order of tuple to stay consistent with the `StandaloneBindle` case 
```suggestion
    RemoteBindle(BindleConnectionInfo, bindle::Id),
```